### PR TITLE
VIX-3179 Create Fixture Property Editor

### PR DESCRIPTION
**Pull request by johncbaur**
VIX-3179 Create Fixture Property Editor

### CODE REVIEW

**Comment by jeffu231** [comment]
Looks like this one needs rebased for the changes in VIX-3076

**Comment by johncbaur** [comment]
Should be rebased now.

**Review by jeffu231 — CHANGES_REQUESTED** [review]
There is a lot to look over here. It looks pretty sensible scanning over it. Just one minor observation that caught my eye.

File: Modules/Editor/FixturePropertyEditor/Views/SelectFixtureSpecificationView.xaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                                <SolidColorBrush x:Key="{x:Static SystemColors.InactiveSelectionHighlightBrushKey}" Color="#FF333333"/>
+                            </Style.Resources>
+                        </Style>
+                    </Setter.Value>
+                </Setter>
+                <Style.Resources>
+                    <SolidColorBrush x:Key="{x:Static SystemColors.HighlightBrushKey}" Color="#808080" />
+                    <SolidColorBrush x:Key="{x:Static SystemColors.ControlBrushKey}" Color="#FF333333" />
+                    <SolidColorBrush x:Key="{x:Static SystemColors.HighlightTextBrushKey}" Color="Black" />
+                    <SolidColorBrush x:Key="{x:Static SystemColors.InactiveSelectionHighlightBrushKey}" Color="#FF333333"/>
```
[the remark this answers is not in the document]
Comment (johncbaur) [reply Modules/Editor/FixturePropertyEditor/Views/SelectFixtureSpecificationView.xaml:52]: I copied that style from LayerEditor generic.xaml.  I am presuming that we should move that style to the Theme.xaml and then try to incorporate the constant colors?  Do you want me to modify the LayerEditor as part of resolving this comment or write a new ticket for that?

[the remark this answers is not in the document]
Comment (jeffu231) [reply Modules/Editor/FixturePropertyEditor/Views/SelectFixtureSpecificationView.xaml:52]: Yes, try to refactor that up to the common theme and correct those places that have copies of it so we can make the code more consistent.

[the remark this answers is not in the document]
Comment (johncbaur) [reply Modules/Editor/FixturePropertyEditor/Views/SelectFixtureSpecificationView.xaml]: I moved the ListBox style to theme.xml and removed it from the two original locations.  I also defined color constants and a brush.  I did not adjust any of the colors as that would change the look and feel.

**Comment by johncbaur** [comment]
I think this ticket / branch is going to now need VIX-3200 and VIX-3201 to compile.

**Review by jeffu231 — APPROVED** [review]

**Pull request merged**